applications/api-cat/src/main/resources/banner.txt

```diff
@@ -0,0 +1,7 @@
+               _                      _
+  __ _  _ __  (_)         ___   __ _ | |_
+ / _` || '_ \ | | _____  / __| / _` || __|
+| (_| || |_) || ||_____|| (__ | (_| || |_
+ \__,_|| .__/ |_|        \___| \__,_| \__|
+       |_|
+${application.formatted-version}, Spring Boot ${spring-boot.formatted-version}, Log level ${logging.level.no.acat}
```

Comment:
Har du sørget for å håndtere banner.txt i pom.xml? Se andre pom.xml i moduler som har banner.txt